recipe/meta.yaml

```diff
@@ -31,12 +31,18 @@ requirements:
     - meson
     - ninja
     - pkg-config
+    - pthread-stubs
   host:
     - cairo
     - freetype
     - glib
     - icu
     - graphite2
+  run:
```

Comment:
You should not need to explicitly specify these run dependencies; they should be automatically injected by `conda build` as `run_exports` from the `host` dependencies.